src/plugins/points/decay.py

```diff
@@ -12,8 +12,8 @@ class DecayPointsPlugin(PointsPlugin):
 
     def get_points(self, team, flag, solves, *args, **kwargs):
         challenge = self.challenge
-        decay_constant = challenge.challenge_metadata["decay_constant"]
-        min_points = challenge.challenge_metadata["min_points"]
+        decay_constant = challenge.flag_metadata.get("decay_constant", 0.99)
+        min_points = challenge.flag_metadata.get("min_points", 100)
```

Comment:
Should we move this into settings?

Reply:
unsure, min_points makes sense to have per challenge imo, because the max points is stored per challenge, decay constant feels like something most people wont need to touch, but if we're giving them control over min and max, it feels a bit odd not giving them control over decay constant

Reply:
No, I mean, Django settings. As in

```suggestion
        decay_constant = challenge.flag_metadata.get("decay_constant", settings.DEFAULT_CHALLENGE_DECAY)
        min_points = challenge.flag_metadata.get("min_points", settings.DEFAULT_CHALLENGE_MINIMUM_POINTS)
```

Reply:
ohhh, ok, that makes sense, i'd either put it there or config, the defaults should definitely be configurable